app/assets/javascripts/course/assessment/submission/answer/answer.js

```diff
@@ -95,9 +119,9 @@
     // TODO: Implement error recovery.
   }
 
-  showCommentToolbar(document);
+  showScriptedWidgets(document);
   $(document).on('DOMNodeInserted', function(e) {
-    showCommentToolbar(e.target);
+    showScriptedWidgets(e.target);
```

Comment:
Actually we might have performance issue if we rely on `DOMNodeInserted` too much.  What I would do is to render a js.erb and show the button there, but it's OK for now.